### ISSUE TITLE
Add publisher/journal: European Respiratory Society

### ISSUE THREAD

**Issue by sarahklippincott** [CDL-Dryad/dryad-product-roadmap#2723]
**Name to appear on membership page:** European Respiratory Society

**Payment plan:** quarterly tiered invoicing plan

**Journals included:** 
European Respiratory Journal
ERJ Open Research
European Respiratory Review
Breathe

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)** 

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)** See [onboarding spreadsheet](https://docs.google.com/spreadsheets/d/1xGT_5T9a73eoDGVARnOpQe1LTOU9jrRUlJVrGJpbuzI/edit#gid=1888868475).

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)** See [onboarding spreadsheet](https://docs.google.com/spreadsheets/d/1xGT_5T9a73eoDGVARnOpQe1LTOU9jrRUlJVrGJpbuzI/edit#gid=1888868475).

**Email contacts to discuss integration of journal metadata with Dryad:** matt.broadhead@ersnet.org

Development tasks:
- [x] add member name to membership page
- [x] add journals
- [x] update payment settings
- [x] update journal notification settings
- [x] add journal administrators
- [ ] start integration process (track on new ticket)
